strawberry/subscriptions/__init__.py

```diff
@@ -1,2 +1,8 @@
+from typing_extensions import Literal
+
 GRAPHQL_TRANSPORT_WS_PROTOCOL = "graphql-transport-ws"
 GRAPHQL_WS_PROTOCOL = "graphql-ws"
+
+SubscriptionProtocolType = Literal[  #  type: ignore
+    GRAPHQL_TRANSPORT_WS_PROTOCOL, GRAPHQL_WS_PROTOCOL  #  type: ignore
+]
```

Comment:
Looking at the pep you linked and after a quick local test, I think it should look like this:

```suggestion
GRAPHQL_TRANSPORT_WS_PROTOCOL = Literal["graphql-transport-ws"]
GRAPHQL_WS_PROTOCOL = Literal["graphql-ws"]

SubscriptionProtocolType = Literal[
    GRAPHQL_TRANSPORT_WS_PROTOCOL, GRAPHQL_WS_PROTOCOL
]
```

Reply:
I removed `SubscriptionProtocolType` as I couldn't get it to work with `subscription_protocols` and `(GRAPHQL_WS_PROTOCOL , GRAPHQL_TRANSPORT_WS_PROTOCOL)` as a default value.